docs/workloads/azure/data/getting_started/de_shared_resourcedeployment_azure.md

```diff
@@ -0,0 +1,64 @@
+---
+id: de_shared_resourcedeployment_azure
```

Comment:
Maybe rename this id (and file name) to `shared_resources_deployment_azure`?

Reply:
updated